pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -2075,6 +2081,13 @@ def test_td64arr_div_numeric_array(self, box_with_array, vector, any_real_dtype)
                 expected = [tdser[n] / vector[n] for n in range(len(tdser))]
             expected = pd.Index(expected)  # do dtype inference
             expected = tm.box_expected(expected, xbox)
+
+            if using_array_manager and box_with_array is pd.DataFrame:
+                # https://github.com/pandas-dev/pandas/issues/39750
+                # third column with all-NaT as result doesn't get preserved
+                # as timedelta64 dtype
+                expected[2] = Series([pd.NaT, pd.NaT], dtype=object)
```

Comment:
this is sub-optimal.  can you add a TODO to try to retain the correct dtype

Reply:
I don't fully understand your comment. Of course it's sub-optimal, as it's a bug (but an existing one, and I opened an issue about it)

Reply:
> but an existing one

now im confused; we only get here if `using_array_manager`

Reply:
i.e. if the expected isnt actually what we want, better to xfail

Reply:
> now im confused; we only get here if using_array_manager

Yes, I "uncovered" the bug by adding array manager tests, but the bug itself is not related to ArrayManager, it's a bug in TimedeltaArray (and you can run into it without ArrayManager as well, just not covered by any test)

But indeed, an xfail might be more appropriate since I am now asserting the buggy behaviour. The problem is that I would then no longer test the other parts of this test (the other columns) which is actually working fine. 
So I would prefer to keep it this way (but will update the comment to make it more clear this is buggy)

Reply:
As mentioned above, and already included in the comment as well, I already opened an issue (it's https://github.com/pandas-dev/pandas/issues/39750)

Reply:
> and you can run into it without ArrayManager as well, just not covered by any test

OK.  Can you add a non-ArrayManager test that will hit it?

Reply:
I added additional examples that don't involve ArrayManager to the issue